File: comscoreanalytics/src/main/java/com/bitmovin/player/integrations/comscoreanalytics/ComScoreConfiguration.java
package com.bitmovin.player.integrations.comscoreanalytics;

public class ComScoreConfiguration {
    private String publisherId;
    private String publisherSecret;
    private String applicationName;

    public ComScoreConfiguration(String publisherId, String publisherSecret, String applicationName) {
        this.publisherId = publisherId;
        this.publisherSecret = publisherSecret;
        this.applicationName = applicationName;
    }

    public String getPublisherId() {
        return publisherId;
    }

    public String getPublisherSecret() {
        return publisherSecret;
    }

    public String getApplicationName() {
        return applicationName;
    }

}

File: comscore/src/main/java/com/bitmovin/player/integration/comscore/ComScoreConfiguration.kt
package com.bitmovin.player.integration.comscore

data class ComScoreConfiguration(
    val publisherId: String,
    val applicationName: String,
    var userConsent: ComScoreUserConsent = ComScoreUserConsent.UNKNOWN,
    val secureTransmission: Boolean = false,
    val childDirectedAppMode: Boolean = false,
    val debug: Boolean = false
)
